docs/root/operations/admin.rst

```diff
@@ -179,8 +179,10 @@ modify different aspects of the server:
 
 .. http:post:: /logging
 
-  Enable/disable different logging levels on different subcomponents. Generally only used during
-  development.
+  Enable/disable different logging levels on a particular logger or all loggers. Generally only used during
```

Comment:
While you're at this, could you help to improve it by using e.g. `.. note::` for the "Generally only used during development" sentence? And optionally put the below use cases into a bulleted list. WDYT?

Reply:
Sounds great, will do. Appreciate you taking the time to provide suggestions.